spec/components/app_consent_response_component_spec.rb

```diff
@@ -30,9 +30,10 @@
   end
 
   context "with consent taken over the phone" do
-    let(:consent) { create(:consent, :given, route: "phone") }
+    let(:consent) { create(:consent, :given_verbal) }
```

Comment:
Just for the readingz

```suggestion
    let(:consent) { create(:consent, :given_verbally) }
```